.github/workflows/ci-publish.yml

```diff
@@ -84,6 +87,6 @@ jobs:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: |
-          python -m pip install --upgrade pip
+          poetry build
           pip install twine
           twine upload dist/*
```

Comment:
I believe poetry replaces the need for twine, so I think we should be using "poetry publish"